src/passes/param-utils.cpp

```diff
@@ -290,6 +290,7 @@ void localizeCallsTo(const std::unordered_set<Name>& callTargets,
                      std::function<void(Function*)> onChange) {
   struct LocalizerPass : public WalkerPass<PostWalker<LocalizerPass>> {
     bool isFunctionParallel() override { return true; }
+    bool requiresNonNullableLocalFixups() override { return false; }
```

Comment:
It would be good to add a comment about why this is true, i.e. `// Always sets locals immediately before using them.`